### ISSUE TITLE
Scid Alias bug in lnd config

### ISSUE THREAD

**Issue by Eternumkr** [mynodebtc/mynode#697]
**Describe the bug**
when adding  protocol.option-scid-alias=true lnd refuses to start due to parsebool syntax error

**Expected behavior**
lnd should run properly enabling scid alias

**MyNode hardware (please complete the following information):**
 - Device: [raspberry pi 4 8gb
 - Version v0.3.01

**Additional context**
my node alias kept popping up in the error. even when i changed my alias for my node the error should be the same but would reflect the change in alias.

**Comment by tehelsper** [comment]
Did you put it in a [protocol] section of the config?

Try checking here:
https://github.com/lightningnetwork/lnd/blob/master/sample-lnd.conf

**Comment by Eternumkr** [comment]
yes. Still gave the same error with or without adding "[Protocol]"

**Comment by Eternumkr** [comment]
Sep 27 20:47:06 myNode lnd[27370]: failed to load config: /home/bitcoin/.lnd/lnd.conf:40: strconv.ParseBool: parsing "My.Alias": invalid syntax 

The above is the specific error. 
My config is below,

# lnd configuration
# /home/bitcoin/.lnd/lnd.conf

[Application Options]
maxpendingchannels=10
alias=Notmyreal.Alias
color=#68F442
bitcoin.feerate=350
bitcoin.basefee=0
nat=false
tlsextradomain=justsomerandomonionaddressnotimportant.onion

# Tor-only by default
#listen=localhost
externalip=xxx.xxx.xx.xx:9735
listen=0.0.0.0:9735

acceptortimeout=30s

rpclisten=0.0.0.0:10009
restlisten=0.0.0.0:10080

tlsextraip=0.0.0.0
tlsextradomain=mynode.local

debuglevel=info
#debuglevel=debug

# This line needs to be uncommented if you have a lnd wallet and commented if you do not
wallet-unlock-password-file=/mnt/hdd/mynode/settings/.lndpw
wallet-unlock-allow-create=true

ignore-historical-gossip-filters=false
accept-keysend=true
accept-amp=true
caches.channel-cache-size=3000000

[protocol]
protocol.wumbo-channels=true
protocol.option-scid-alias=true
protocol.no-anchors=true

[Bitcoin]
bitcoin.active=1

# enable either testnet or mainnet
bitcoin.mainnet=1
bitcoin.testnet=0

bitcoin.node=bitcoind

[Bitcoind]
bitcoind.rpchost=127.0.0.1:8332

[autopilot]
#autopilot.active=1
#autopilot.maxchannels=5
#autopilot.allocation=0.6

[bolt]
db.bolt.auto-compact=true

[routing]
routing.strictgraphpruning=false


[Tor]
tor.active=true
tor.v3=true
tor.streamisolation=false
tor.skip-proxy-for-clearnet-targets=true

**Comment by Eternumkr** [comment]
The fix for this issue is to do the following,

You need to ssh into your node and go to /home/bitcoin/.lnd/lnd.conf and change "protocol.option-scid-alias=*FOR SOME REASON THIS CHANGES TO YOUR NODE ALIAS SO THATS WHY LND THROWS A FIT" to protocol.option-scid-alias=true
